src/main/resources/openapi/transcriptions.yaml

```diff
@@ -1706,6 +1720,68 @@ components:
           type: string
           description: 'The comments for the admin action'
 
+    TranscriptionResponseDetails:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: 'The transcription document id'
+
+    CaseResponseDetails:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: 'The case id'
+        case_number:
+          type: string
+          description: 'The case number'
+
+    CourthouseResponseDetails:
+      type: object
+      properties:
+        id:
+          type: integer
+        display_name:
+          type: string
+
+    CourtroomResponseDetails:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+
+    HearingResponseDetails:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: 'The case id'
+        hearing_date:
+          type: date
+          description: 'The case number'
+
+    AdminMarkedForDeletionResponse:
```

Comment:
as the response is a list, this object does not represent teh whole response, but just an item in it, so might be better to call it `AdminMarkedForDeletionResponseItem`